sdcm/remote/remote_long_running.py

```diff
@@ -0,0 +1,57 @@
+import time
+import logging
+from uuid import uuid4
+from pathlib import Path
+from contextlib import ExitStack
+
+from sdcm.remote.remote_cmd_runner import RemoteCmdRunnerBase
+from sdcm.remote.libssh2_client.result import Result
+from sdcm.remote.libssh2_client.exceptions import UnexpectedExit
+
+logger = logging.getLogger(__name__)
+
+
+def run_long_running_cmd(remoter: RemoteCmdRunnerBase,
+                         cmd: str,
+                         timeout: float | None = None,
+                         ignore_status: bool = False,
+                         verbose: bool = True,
+                         retry: int = 1):
+
+    cmd_uuid = uuid4()
+    cmd_stdout_log = Path('/tmp') / f'remoter_stdout_{cmd_uuid}.log'
+    cmd_stderr_log = Path('/tmp') / f'remoter_stderr_{cmd_uuid}.log'
+    cmd_exit_log = Path('/tmp') / f'remoter_exit_{cmd_uuid}.log'
+    cmd_bash = Path('/tmp') / f'remoter_cmd_{cmd_uuid}.sh'
+
+    with ExitStack() as stack:
+        def clear_tmp_files():
+            for log_file in (cmd_stdout_log, cmd_stderr_log, cmd_exit_log, cmd_bash):
+                remoter.run(f'rm {log_file}', verbose=False, ignore_status=True)
+
+        stack.callback(clear_tmp_files)
+        remoter.run(f'echo "{cmd}" > {cmd_bash}')
+
+        start_time = time.perf_counter()
+
+        remote_command = f'(bash +e +x -o pipefail {cmd_bash} ; echo $? > {cmd_exit_log}) >& {cmd_stdout_log} 2> {cmd_stderr_log} </dev/null  &  jobs -p'
```

Comment:
`+e +x` - shouldn't it be `-e -x`?

Reply:
we can enable it on verbose maybe, but we don't want base debug
and we don't want to fail on the first failure, i.e. depends what's the command being passed in, i.e. the user of this code, can enable/disable those flag  as needed.

Reply:
removing those, since those are bash defaults anyhow